WINDOWS_LINUX_SUBSYSTEM.md

```diff
@@ -12,6 +12,32 @@ The following instructions will help you to get ready for [Le Wagon](http://www.
 - Install Ruby
 
 
+## Remote tools
+
+To be able to interact when you are not on site, we will be using two tools.
+
+### Zoom
+
+Zoom is a video conferencing tool, go to [Zoom page](https://zoom.us/download) and under **Zoom Client for Meetings** hit the **Download** button. Open the file you have just downloaded. A progress bar will appear, then Zoom will start. Click on **Connexion** and create an account. Once connected, Zoom should look like this:
+
+![remote_zoom.png](images/remote_zoom.png)
+
+You can close Zoom now :wink:.
+
+### Teamviewer
+
+For the most complicated problems, a teacher might have to take control of your computer. To be able to do this, we will need to use the Teamviewer tool. Go to the [Teamviewer download page](https://www.teamviewer.com/en/download). It should automatically detect your operating system. if it doesn't, choose your operating system from the list at the top of the page. Click on **Download Teamviewer**. Open the file you just have downloaded. Leave the default settings and click on **Accept**. A progress bar will appear, then Teamviewer will start when the installation is over. It should look like this:
+
+![teamviewer.jpg](images/teamviewer.jpg)
+
+You can close Teamviewer now :wink:.
+
+:warning: We will not, and cannot take control of your computer without your permission. Your Teamviewer password changes every time you restart the application :thumbsup:. We will only use this tool as a last resort. :warning:
```

Comment:
Let's remove the ⚠️ which doesn't feel reassuring, and shorten the text a bit:

**N.B**: This will only be used as last resort when debugging becomes too tricky orally. Nobody will ever be able to take control of your screen without you knowing it 👌